plugins/module_utils/elbv2.py

```diff
@@ -107,6 +109,23 @@ def get_elb_attributes(self):
         # Replace '.' with '_' in attribute key names to make it more Ansibley
         return dict((k.replace('.', '_'), v) for k, v in elb_attributes.items())
 
+    def get_elb_ip_address_type(self):
+        """
+        Retrieve load balancer ip address type using describe_load_balancers
+
+        :return:
+        """
+
+        try:
+            self.elb_ip_addr_type = AWSRetry.jittered_backoff()(
+                self.connection.describe_load_balancers
+            )(LoadBalancerArns=[ self.elb['LoadBalancerArn'] ])['LoadBalancers'][0]['IpAddressType']
+            return self.elb_ip_addr_type
+
+        except (BotoCoreError, ClientError) as e:
+            self.module.fail_json_aws(e)
```

Comment:
```suggestion
        return self.elb.get('IpAddressType', None)
```

We already have the result of `describe_load_balancers(...)['LoadBalancers'][0] in self.elb (initialized in __init__  a couple of line above the get_elb_ip_address_type call